include/smb2/smb2-errors.h

```diff
@@ -19,11 +19,12 @@
 /*
  * NTSTATUS fields
  */
-#define SMB2_STATUS_SEVERITY_MASK    0xc0000000
-#define SMB2_STATUS_SEVERITY_SUCCESS 0x00000000
-#define SMB2_STATUS_SEVERITY_INFO    0x40000000
-#define SMB2_STATUS_SEVERITY_WARNING 0x80000000
-#define SMB2_STATUS_SEVERITY_ERROR   0xc0000000
+#define SMB2_STATUS_ERROR_SEVERITY_MASK    0xc0000000
+#define SMB2_STATUS_WARNING_SEVERITY_MASK  0x80000000
```

Comment:
No, leave it as it was before.
The severity bitmask is the top two bits and it encods 4 different values,  success/info/warning and error.

So the way to test if something is an error is thus first to mask out the two severity bits and then check what these two bits are,
for example

switch (status & SMB2_STATUS_SEVERITY_MASK) {
case SMB2_STATUS_SEVERITY_SUCCESS:  printf("success!\n"); break;
case SMB2_STATUS_SEVERITY_ERROR: printf("was error\n"); break;
...

Reply:
done.